packages/manager/modules/vps/src/upscale/upscale.constants.js

```diff
@@ -12,6 +13,9 @@ export const RANGES = {
   ELITE: 'Elite',
   STARTER: 'Starter',
   VALUE: 'Value',
+  LE_16: 'le-16',
```

Comment:
Are these values used ? If not you can remove them from `RANGES` I guess

Reply:
Yes these values are uses in different locations of the code.